soroban-sdk/src/lib.rs

```diff
@@ -802,6 +802,8 @@ pub mod data {
 }
 pub mod auth;
 mod bytes;
+#[cfg(any(test, feature = "testutils"))]
+pub mod cost_estimate;
```

Comment:
Move this into the testutils directory and you wo'nt need a cfg testutils here.

Also note that whenever we cfg testutils, we also need to cfg_attr a doc attribute to tell the docs generation that it's only usable in testutils. You won't need to do that if you move it into the testutils directory, but just a knowledge share.

Reply:
Acknowledged.